cmd/ztest/ztest.c

```diff
@@ -123,7 +123,7 @@
 #include <math.h>
 #include <sys/fs/zfs.h>
 #include <libnvpair.h>
-#ifdef __GNUC__
+#ifdef __GLIBC__
```

Comment:
`execinfo.h` and `backtrace()` are GNU extensions provided by *glibc* and not by *gcc* so the change is correct.

The mentioned [gcc documentation](https://gcc.gnu.org/onlinedocs/gcc/C-Extensions.html) does not mention backtrace() or execinfo.h as far I can see.

An alternative way to solve it would be to do a configure check for it and use `HAVE_EXECINFO_H` and `HAVE_BACKTRACE`.